p4-fusion/git_api.cc

```diff
@@ -322,15 +323,16 @@ std::string GitAPI::WriteChangelistBranch(
 
 	for (auto& file : files)
 	{
+		auto relativePath = decodePath(file.GetRelativePath());
 		if (file.IsDeleted())
 		{
-			checkGit2Error(git_index_remove_bypath(idx, file.GetRelativePath().c_str()));
+			checkGit2Error(git_index_remove_bypath(idx, relativePath.c_str()));
 		}
 		else
 		{
 			git_index_entry entry = {
 				.mode = GIT_FILEMODE_BLOB,
-				.path = file.GetRelativePath().c_str(),
+				.path = relativePath.c_str(),
 			};
```

Comment:
could file.GetRelativePath do that decode, so it can't be forgotten nd we don't have to import p4 into a file called git? :)